### PR TITLE
DIP-210 and DIP-212 multihash for Announcements and Activity Content

### PR DESCRIPTION
**Pull request by wesbiggs**
Problem
=======
DSNP Announcements and the "hash" addition to Activity Content should use the multihash format for content hashes to avoid algorithm lock-in.

Link to GitHub Issue(s): #210, #212

Solution
========
All Announcements that have a `contentHash` have been updated, and the section on validation of Announcements is extended to include a list of supported hashing algorithms.

The description and examples using the`"hash"` JSON property with Activity Content documents have been updated.

The supported hashing algorithms for this version are Blake2b and SHA-256.

Change summary:
---------------
- [ ] Updated spec pages for Announcements
- [ ] Updated spec pages for Activity Content

Steps to Verify:
----------------
1. Generate markdown and review

### CODE REVIEW

File: pages/DSNP/Announcements.md [hunk 1]
```diff
@@ -27,6 +21,26 @@ Each Announcement has an enumerated type for use when separating out a stream of
...
+The content hash MUST be generated by applying a Supported Hashing Algorithm to the full, unaltered contents retrieved from the URL.
+When readers retrieve content referenced in an Announcement, they can validate the authenticity of the content by regenerating the hash output and comparing it with the content hash recorded in the Announcement.
+
+DSNP Announcements encode content hashes using the [multihash](https://github.com/multiformats/multihash) specification.
+
+### Supported Hashing Algorithms
+
+| Algorithm | Multihash Name | Leading bytes (as varint) | Reference | DSNP Version Added |
+| --- | --- | --- | --- | --- |
+| BLAKE2b | `blake2b-256` | `0xa0e40220` | [RFC 7693](https://tools.ietf.org/html/rfc7693) | 1.2.0 |
```
Comment (wesbiggs) [inline pages/DSNP/Announcements.md:42]: Let's see what the IPFS default multihash algorithm is

**Review by wilwade — CHANGES_REQUESTED** [review]
Looks good. One addition: needs to be added to the changelog

**Review by shannonwells — APPROVED** [review]
nothing to add, all good from my end

**Comment by wesbiggs** [comment]
I added the Activity Content changes to this same PR.

File: pages/ActivityContent/Associated/Hash.md [hunk 2]
```diff
@@ -3,34 +3,27 @@
...
-    {
-      "algorithm": "keccak256",
-      "value": "0x1234567890ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF"
-    },
-    {
-      "algorithm": "Ripemd256",
-      "value": "0x96a9209006748794193d1811ef2dd5f447782b8b1635841165bc031bb3db64da"
-    }
+    "QmQNHNfHnbgJJ6nK4UPx2VtTUCafAKCbqZJ6ZRYUGjoeFj",
+	"2DrjgbGgSsXRhTiBWckoVwBFC6H4qiBWWNumSsRwdUt82YnTdN"
```
Comment (wilwade) [inline pages/ActivityContent/Associated/Hash.md:26]: Whitespace
```suggestion
	"QmQNHNfHnbgJJ6nK4UPx2VtTUCafAKCbqZJ6ZRYUGjoeFj",
	"2DrjgbGgSsXRhTiBWckoVwBFC6H4qiBWWNumSsRwdUt82YnTdN"
```

File: pages/DSNP/Types/Profile.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ The reference content *MUST be of profile type*.
 | Field | Description | Serialization | Parquet Type | Bloom Filter |
```
Comment (wilwade) [inline pages/DSNP/Types/Profile.md:8]: I only just noticed that `Profile` doesn't have a `Data Type` column. I think this was just an oversight. I'm ok with fixing in this or a different PR.

**Review by wilwade — APPROVED** [review]
One whitespace and something unrelated. Nothing blocking.

**Pull request merged**